### PR TITLE
Capture elapsed time for each HAR entry

### PR DESCRIPTION
**Pull request by thatplguy**
Tested manually with `akibox-flask`: https://github.com/akitasoftware/akibox-flask/pull/2

### CODE REVIEW

File: akita_flask/testing.py [hunk 1]
```diff
@@ -90,13 +90,19 @@ def wsgi_to_har_entry(start: datetime, request: Request, response: Response) ->
         bodySize=len(content),
     )
 
+    # datetime.timedelta doesn't have a total_milliseconds() method,
+    # so we compute it manually.
+    elapsed_time = datetime.now(timezone.utc) - start
+    elapsed_time_ms = ((elapsed_time.days * 86400 + elapsed_time.seconds) * 10**6 +
+                       elapsed_time.microseconds) / 10**3
```
Comment (liujed) [inline akita_flask/testing.py]: How about something like `(datetime.now(timezone.utc).timestamp() - start.timestamp()) * 1000`?

Comment (liujed) [reply akita_flask/testing.py]: Better yet, `elapsed_time = (datetime.now(timezone.utc) - start) / timedelta(milliseconds=1)`.

Comment (thatplguy) [reply akita_flask/testing.py]: Sure, fixed in 26e547c.

**Review by liujed — APPROVED** [review]
LGTM!

**Pull request merged**